tensorflow/workspace.bzl

```diff
@@ -508,6 +508,14 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   )
 
   native.new_http_archive(
+    name = "lmdb",
```

Comment:
Thanks for looking into this, @jart! Not sure why Jenkins complains about license.
```
FAIL: mismatch in packaged licenses and external dependencies
Missing the licenses for the following external dependencies:
@lmdb//
```